apps/explorer/src/hooks/useMedia.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { useImageMod } from './useImageMod';
+
+export function useMedia(url: string) {
+    const formatted = url.replace(/^ipfs:\/\//, 'https://ipfs.io/ipfs/');
+    const { data: allowed } = useImageMod({ url: formatted });
+    return { url: formatted, nsfw: !allowed };
```

Comment:
Can you default nsfw to true for a small period of time to allow the image mod service time to respond, so that we don't flash potentially sensitive content on the page while we wait for a response?